examples/virtio/include/serial_config.h

```diff
@@ -6,7 +6,7 @@
 #include <stdint.h>
 
 /* Number of clients */
-#define SERIAL_NUM_CLIENTS 3
+// #define SERIAL_NUM_CLIENTS 3
```

Comment:
Why this is commented out now?

Reply:
Oh my bad that was before we made `NUM_CLIENTS` consistent

Reply:
Done